rfc8446.xml

```diff
@@ -2770,6 +2770,9 @@ information).</t>
 above-mentioned criteria (in addition to other criteria, such as
 transport-layer endpoint, local configuration, and preferences).</t>
 
+<t>Note that there are certificates that use algorithm combinations that
+cannot currently be used with TLS.</t>
```

Comment:
This seems hard to understand in isolation.

Reply:
The story here (which apparently didn't make it into my revew?  Must not have pressed 'save') is that in the client-certificate-selection section, we have a "Note that, as with the server certificate, there are certificates that[...]", which feels like a dangling reference if we don't have something like this.
Removing that other note might be a better plan; I didn't think too hard.

Reply:
I just feel that absent an example this is too hard to read. Suggest we remove both

Reply:
Removing both works for me

Reply:
Done.